content/hardware/08.edu/solution-and-kits/alvik/tutorials/setting-alvik-arduino-ide/setting-alvik-arduino-ide.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "Setting up Alvik on Arduino IDE"
+difficulty: beginner
+description: "Learn how to configure your Alvik for Arduino IDE"
+tags:
+  - Robot, MicroPython, Education
+author: "Pedro Sousa Lima"
+---
+The Arduino® Alvik robot was designed to be compatible with both C++ and MicroPython. To ensure a smooth experience, functions have the same structure and parameters across all environments. This means the [Alvik's API reference](https://docs.arduino.cc/tutorials/alvik/api-overview/) can be used as a resource regardless of which environment you choose. In this guide, we will prepare the board to be programmed using the Arduino IDE.
+
+## Requirements
+
+### Software
+
+- **Arduino IDE**: A modern desktop-based [Arduino IDE](https://support.arduino.cc/hc/en-us/articles/360019833020-Download-and-install-Arduino-IDE).
+- **Alvik Library for Arduino**: A [library](https://github.com/arduino-libraries/Arduino_Alvik) that provides easy access to the Alvik robot's functionalities.
+- **USB Drivers**: Ensure you have the correct drivers installed to communicate with Alvik via USB.
+- **STM32Cube**: You will need this 3rd party programmer tool by ST available [here](https://www.st.com/en/development-tools/stm32cubeprog.html) to program the STM board.
```

Comment:
```suggestion
```

This tutorial doesn't use STM32CubeProgrammer in any way. It is an unpleasant experience get the software from the STMicroelectronics website because they require you to sign up for an account and ask for a lot of personal information. So I think it is a bad idea to give the user the impression they must install STM32CubeProgrammer before they can follow the tutorial.

Reply:
Thanks @per1234 this is a ghost from an old version of the document. Fixed.